src/components/Legend.js

```diff
@@ -106,6 +106,16 @@ class Legend extends Component {
     ));
   }
 
+  getLegendCssClass(iframeModeActive, isMobile) {
+    if (iframeModeActive === true) {
```

Comment:
`=== true` is hier overbodig, gewoon `if (iframeModeActive)` doen.

Reply:
Is idd korter. Ik ga het aanpassen. Ik zie dat het ook op andere plaatsen in de code zo wordt gechecked. De reden dat ik hier `===` gebruikte is omdat er dan specifiek op een boolean wordt gecheckt. `if ('false')` is bijvoorbeeld truthy.